test/lib/cql_test_env.cc

```diff
@@ -944,6 +944,10 @@ class single_node_cql_env : public cql_test_env {
             with_scheduling_group(dbcfg.statement_scheduling_group, [&func, this] {
```

Comment:
> Before this patch, there was a 1s sleep on gossiper shutdown, which I
don't know where it comes from. After the patch there is no such
sleep.

The frequency of main gossiper loop is once per second
```
    static constexpr std::chrono::milliseconds INTERVAL{1000};
```

maybe we're waiting for the last gossiper loop iteration to execute?

Reply:
likely